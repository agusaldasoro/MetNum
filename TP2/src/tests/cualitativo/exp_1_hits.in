1 -1 0  tests/cualitativo/exp_1.txt 0.00001