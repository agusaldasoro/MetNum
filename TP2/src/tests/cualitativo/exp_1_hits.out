0.495382
0.608216
0.279821
0.140172
0.361404
0.171611
0.000000
0.171611
0.000000
0.000000
0.311783
0.000000
0.000000
0.000000
0.000000
0.571689
0.148704
0.000000
0.439770
0.466951
0.182577
0.182577
0.182577
0.148704
0.148704
0.148704
0.148704
0.083999
0.083999
0.192488
